examples/helloworld/src/index.js

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import _ from 'underscore';
 import App from './App';
 
-React.render(<App />, document.getElementById('root'))
+
+var state = {
+    very: {
+        deeply: {
+            nested: {
+                counts: _.range(400).map(function () { return 0; })
+            }
+        }
+    }
+};
+
+function swapper(f) {
```

Comment:
[Object.observe can be polyfilled in all browsers with a polling approach](https://github.com/MaxArt2501/object-observe), but it is nondeterministic

Here is a nice and tiny js implementation of atom concept: [https://github.com/cjohansen/js-atom](cjohansen/js-atom)

I like the atom approach better than the Cursor.render approach.